docs/src/cli/transfer-tokens.md

```diff
@@ -98,6 +98,39 @@ solana balance <ACCOUNT_PUBKEY> --url http://devnet.solana.com
 where `<PUBKEY>` is either the public key from your keypair or the
 recipient's public key.
 
+## Set a default fee-payer
+
+Every transaction includes a fee, which is paid from a fee-payer account.
+The fee-payer is specified with `--fee-payer=<KEYPAIR>`. If the option,
+is missing, commands use the keypair at `--keypair=<KEYPAIR>` as the
+fee-payer. If that option is missing as well, commands use the *default signer*,
+which is set in your CLI configuration:
+
+```bash
+solana config set --keypair=<KEYPAIR>
+```
+
+For convenience, consider setting the default signer to the account you want
+to pay transaction fees. Because transaction fees are so inexpensive on
+Solana, it is oftentimes sufficient to use a keypair file for this. Although
+keypair files are the most insecure way to store tokens, a fee-payer account
+rarely needs to hold more than a fraction of a SOL.
+
+Here is how to create and configure a fee-payer account:
+
+```bash
+solana-keygen --no-passphrase -o ~/fee-payer.json
+solana config set --keypair=~/fee-payer.json
+```
+
+Now transfer it a small amount of SOL:
+
+```bash
+solana transfer --fee-payer=<SENDER_KEYPAIR> --from=<SENDER_KEYPAIR> ~/fee-payer.json 1
+```
+
+Hopefully that is the last time you use the `--fee-payer` option!
```

Comment:
Not sure we want to encourage people not to use `--fee-payer` with such gusto.  `--keypair` is already dangerously overloaded as the default authority for everything to be used as a casual hot key.  I know this sentence feels nice to us because of our experiences in toiling through the CLI growing pains, but I don't think we should discourage explicitness.

Reply:
I suppose what we really need is `solana config set --fee-payer=<KEYPAIR>`

Reply:
Ripped it out